package.json

```diff
@@ -1,17 +1,19 @@
 {
   "name": "@buildium/angular-elements",
-  "version": "1.2.0-0",
+  "version": "1.2.0-1",
   "description": "A collection of directives for building angular applications",
   "main": "lib/index.js",
   "scripts": {
+    "pretest": "eslint src/",
     "test": "karma start --single-run",
     "compile": "babel src --out-dir lib/ --presets=es2015",
     "dist": "mkdirp dist && browserify lib/index.js -o dist/buildium-angular-elements.js",
     "build": "npm run compile && npm run dist",
-    "prepublish": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
on second thought (_based on some offline review_), we still want this post-install since `compile` could fail, which would only get caught on a release build